themes/cdap/assets/js/get-started.js

```diff
@@ -21,7 +21,7 @@ const dropdownProductionItems = document.querySelectorAll('.production-nav-js');
 const dropDownCta = document.querySelector('.dropdown-toggle');
 const panesEvaluation = document.querySelectorAll('.evaluation-wrapper-js');
 const panesProduction = document.querySelectorAll('.production-wrapper-js');
-let persistBtnText = dropDownCta.innerText.trim();
+let persistBtnText = dropDownCta ? dropDownCta.innerText.trim() : '';
```

Comment:
when will ``dropDownCta`` be ``null``?

Reply:
It will be null when there are no dropdowns on the page.